### PR TITLE
app-crypt/nitrocli: Bump version to 0.2.1

### PR DESCRIPTION
**Pull request by d-e-s-o**
This patch adds an ebuild for app-crypt/nitrocli version 0.2.1. The
application includes a few new subcommands and dependencies have been
updated.

Signed-off-by: Daniel Müller <deso@posteo.net>

### CODE REVIEW

**Comment by d-e-s-o** [comment]
@gyakovlev given that you should have the most context, mind taking a look?

**Comment by gyakovlev** [comment]
here are the changes I'm referring to:
https://github.com/gentoo/gentoo/commit/3a324245485e1430f779c316315ce367886cb594
https://github.com/gentoo/gentoo/commit/df05e92b7444fee1caf052a9ac753b24aa34f070

File: app-crypt/nitrocli/nitrocli-0.2.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# We require gnupg for /usr/bin/gpg-connect-agent.
+RDEPEND="
+	${DEPEND}
+	app-crypt/gnupg
+"
+
+QA_FLAGS_IGNORED="/usr/bin/nitrocli"
+
+src_install() {
+	cargo install -j $(makeopts_jobs) --path=. --root="${D}/usr" $(usex debug --debug "") \
```
[the remark this answers is not in the document]
Comment (d-e-s-o) [reply app-crypt/nitrocli/nitrocli-0.2.1.ebuild]: Done.

File: app-crypt/nitrocli/nitrocli-0.2.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		|| die "cargo install failed"
+	rm "${D}/usr/.crates.toml" || die "failed to remove .crates.toml"
```
[the remark this answers is not in the document]
Comment (d-e-s-o) [reply app-crypt/nitrocli/nitrocli-0.2.1.ebuild]: Done.

File: app-crypt/nitrocli/nitrocli-0.2.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+	einstalldocs
+	doman "${S}/doc/nitrocli.1"
+}
+
+src_test() {
```
[the remark this answers is not in the document]
Comment (d-e-s-o) [reply app-crypt/nitrocli/nitrocli-0.2.1.ebuild]: Done as well!

**Comment by d-e-s-o** [comment]
Thanks for getting the `eclass` changes merged. Very useful. I've addressed all your comments.

**Comment by gyakovlev** [comment]
tested and pushed, thanks.

btw, do you happen to know is pre `edition = "2018"` cargo behaviour is identical to passing `--path .` ?

I'm thinking of just passing `--path .` in cargo_src_install` just like it passes `root`.
and wondering will it break older ebuilds.

**Comment by d-e-s-o** [comment]
Awesome, thank you!

> btw, do you happen to know is pre `edition = "2018"` cargo behaviour is identical to passing `--path .` ?

Unfortunately I do not know the answer to this question.

**Pull request closed without merging**